core/route_parser.go

```diff
@@ -11,7 +11,8 @@ type route struct {
 }
 
 var routes = []route{
-
+	// https://docs.docker.com/reference/api/docker_remote_api_v1.21/#build-image-from-a-dockerfile
+	{pattern: "/build", method: "POST", action: ActionImageBuild},
```

Comment:
Can you please add this API in the UT?

Reply:
[added](https://github.com/twistlock/authz/pull/48/files#diff-e383ff40d0872136a71e0c9d96b0fafbR17) :-)